configure.ac

```diff
@@ -270,6 +291,17 @@ fi
 [enable_ronn="1"]
 )
 
+AC_SUBST([enable_doc])
+AM_CONDITIONAL([ENABLE_DOC], [test "x$enable_doc" = "x1"])
```

Comment:
This is leftover old code.  It must be from a bad rebase.  I think it can be removed.

Reply:
Agreed, this isn't an intentional inclusion.